schedule/yast/iscsi/iscsi_normal_auth_backstore_fileio_reg.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: iscsi_normal_auth_backstore_fileio
```

Comment:
keep the name same as yaml file name. So for other files.

Reply:
Changed && thanks!